src/main/java/com/tstepnik/planner/exceptions/ExpiredTaskException.java

```diff
@@ -0,0 +1,7 @@
+package com.tstepnik.planner.exceptions;
+
+public class ExpiredTaskException extends RuntimeException {
```

Comment:
```suggestion
public class TaskExpiredException extends RuntimeException {
```

Reply:
Why do we even need this exception?

Reply:
I needed it for my validator 😅

Reply:
It's familiar for client in case he wanted to update task which is expired and he forgot why he cannot do that.